src/main/java/com/fundation/webservice/model/KeyFrameVideo.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.webservice.model;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+
+/**
+ * Implements KeyframeVideo class an convert Method.
+ *
+ * @author Josue Rodriguez
+ * @version 1.0
+ */
+public class KeyFrameVideo {
+    CriteriaKeyFrameVideo criteria;
+
+    public KeyFrameVideo(CriteriaKeyFrameVideo criteria) {
+        this.criteria = criteria;
+    }
+
+    //The convert method add a keyframe for every certain amount of frame.
+    public void convert() {
+        try {
+            String cmd = "ffmpeg -i "
+                    + criteria.getSrcPath()
```

Comment:
Please leave 4-space indentation instead of 8-space.
While we agreed on leaving 8-space-tab for the implementation of methods, it is not the same when invoking a method or doing an assignation, in the latter case we agreed on leaving a 4-space-tab.